syntax = "proto3";

package ray.rpc;

import "src/ray/protobuf/gcs.proto";

message GcsJobInfo {
}

message AddJobReply {
}

message FinishedJob {
}

message MarkJobFinishedReply {
}

// Service for job info access.
service JobInfoAccessService {
  // Add job to gcs server.
  rpc AddJob(GcsJobInfo) returns (AddJobReply);
  // Mark job as finished to gcs server.
  rpc MarkJobFinished(FinishedJob) returns (MarkJobFinishedReply);
}
